shaarli_client/main.py

```diff
@@ -0,0 +1,45 @@
+"""shaarli-client main CLI entrypoint"""
+import json
+from argparse import ArgumentParser
+
+from .v1 import ShaarliV1Client
+
+
+def main():
+    """Main CLI entrypoint"""
+    parser = ArgumentParser()
+    parser.add_argument(
+        'endpoint',
+        choices=['info'],
+        default='info',
```

Comment:
=> #7, each endpoint will be mapped to a parser subcommand, and global help will be displayed if no endpoint is specified

this will be introduced in the next PR, along support with `api/v1/links`